### PR TITLE
Fix #169856 `Dutch domains starting with "A"`

### LINKED ISSUE #169856: Ads on Dutch domains starting with an "A"
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard Browser Extension, Other ad blocker

### AdGuard version

4.2.241

### What type of problem have you encountered?

Missed ads or ad leftovers

### Which browser(s) do you use?

Firefox

### Which device do you use?

Desktop

### Where is the problem encountered?

See comments

### Ad Blocking

EasyList

### Privacy

EasyPrivacy, Fanboy's Enhanced Tracking List

### Social Widgets

_No response_

### Annoyances

_No response_

### Security

_No response_

### Other

_No response_

### DNS filters

_No response_

### Language-specific

AdGuard Dutch filter, EasyList Dutch

### What Stealth Mode options do you have enabled?

_No response_

### Add your comment and screenshots

The normal EasyList Dutch filter list is outdated a lot and in the past they wouldn't approve my new rules as it didn't meet their syntax. But that really hindered me, as some rules need the advanced Adguard/uBlock syntax. 

Below is a list of domains, so you can verify my rules. I have tested them recently (today and max 1 week old):

<details><summary>List with domains where you can find ads or left-overs</summary>

```
https://www.112achterhoek-nieuws.nl/
https://www.402online.com/

7sur7.be, ad.nl, bd.nl, bndestem.nl, destentor.nl, ed.nl, gelderlander.nl, hln.be, pzc.nl, tubantia.nl --> all on homepage

https://adultwebmaster.nl/forums/

https://www.afkortingen.nu/bios & other 2 on https://www.afkortingen.nu/

https://www.ajax1.nl/

https://www.alkmaarguardians.nl/

https://www.alle-tests.nl/ & https://www.bloovi.be/

https://www.alblasserdamsnieuws.nl/

https://allecijfers.nl/verkiezingsuitslagen/

1st: https://almere-nieuws.nl/p2000/
2nd: https://www.almere-nieuws.nl/nieuws/25595/mogelijk-verkeersconflict-voorafgaand-aan-schietpartij-in-tussen-de-vaarten/
3-5: https://www.almere-nieuws.nl/ (& [NSFW] https://www.sexjobs.nl/)

https://ans-online.nl/

https://www.apparata.nl/

https://autosport.nl/

1st: https://www.autowereld.nl/goedkope-occasions
2nd: https://www.autowereld.nl/

https://www.arenalokaal.nl/
https://www.dehavenloods.nl/
https://www.dekrantnieuws.nl/
https://www.destreekkrant.nu/
https://www.gooieneembode.nl/
https://www.hetkrantje-online.nl/
https://www.laardercourant.nl/
https://www.leiderdorpsweekblad.nl/
https://www.nnp.nl/
https://www.weespernieuws.nl/
https://www.zuidenvelder.info/

`.bannergroup`: all on homepage & https://www.dolcevia.com/nl/italie-magazine/lifestyle/4779-kerstavond-in-italie-het-feest-van-de-zeven-vissen

https://www.autowereld.nl/goedkope-occasions
https://www.autowereld.nl/
```
</details>

If I missed an URL then the element is on the homepage.

<details>

```adblock
112achterhoek-nieuws.nl###sidebar
402online.com##.bannerblock
7sur7.be,ad.nl,bd.nl,bndestem.nl,destentor.nl,ed.nl,gelderlander.nl,hln.be,pzc.nl,tubantia.nl##.dfp-space
7sur7.be,ad.nl,bd.nl,bndestem.nl,destentor.nl,ed.nl,hln.be,pzc.nl##.page-header__section--spacer
7sur7.be,ad.nl,bd.nl,bndestem.nl,destentor.nl,ed.nl,gelderlander.nl,hln.be,pzc.nl,tubantia.nl#?#.tile:has(:scope >.ankeiler--advertisement)
adultwebmaster.nl##.main-promo-bn-responsive
afkortingen.nu###banner_rectangle
afkortingen.nu###banner_right
afkortingen.nu###banner_top
agconnect.nl##[class$="_ad-slot"]
alblasserdamsnieuws.nl##.albla-adlabel
alblasserdamsnieuws.nl##.albla-widget
alblasserdamsnieuws.nl#?#.adv-link:not([href*="alblasserdamsnieuws.nl"]):upward(div[id^="albl"])
alblasserdamsnieuws.nl#?#.sidebarbox > .widgettitle:has-text(Banner)
alkmaarguardians.nl###footer
alle-tests.nl,bloovi.be###advertising
almere-nieuws.nl###containerx
almere-nieuws.nl#?#.advertisement_block:upward(1)
almere-nieuws.nl#?#.darksection:has-text(/Door onze partners/i):has(.postbox.owl-carousel)
almere-nieuws.nl#?#.promocentered:upward(section)
almere-nieuws.nl,sexjobs.nl##div[id^="banner-"]
androidworld.nl,cinenews.be,handbal.nl,openrotterdam.nl,webmastersunited.com,wegdamnieuws.nl##.ads
ans-online.nl#?#.theiaStickySidebar > .widget:has-text(/Advertentie\b/)
antilliaansdagblad.com,dolcevia.com,h2owaternetwerk.nl,lokaleomroepzeewolde.nl,radiolelystad.nl##.bannergroup
apparata.nl#?#.widget:has(h2:has-text(/Partners/i))
arenalokaal.nl,dehavenloods.nl,dekrantnieuws.nl,destreekkrant.nu,gooieneembode.nl,hetkrantje-online.nl,laardercourant.nl,leiderdorpsweekblad.nl,nnp.nl,weespernieuws.nl,zuidenvelder.info#?#.component__pubble-banner:has([advobject])
arenalokaal.nl#?#.content-start > * > div[style]:has-text(/Externe links|Externe websites/i)
arenalokaal.nl,dehavenloods.nl,hetkrantje-online.nl,laardercourant.nl,leiderdorpsweekblad.nl#?#.component__pubble-banner:has([advobject]):upward(.bg-gray-100):has-text(/Uit de krant/i)
autosport.nl#?#.active.first:style(margin-top: 0px !important;)
autosport.nl##.advertentie_links
autosport.nl##.bigbanner
autowereld.nl###banner-lister-top
autowereld.nl###mainbanner
```

</details>

### Privacy

- [X] I agree to follow this condition

### PR DESCRIPTION
**Pull request by JohnyP36**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

## What problem does the pull request fix?

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

## What issue is being fixed?
 #169856 
### Add your comment and screenshots

1. Your comment

The normal EasyList Dutch filter list is outdated a lot and in the past they wouldn't approve my new rules as it didn't meet their syntax. But that really hindered me, as some rules need the advanced Adguard/uBlock syntax. 

Below is a list of domains, so you can verify my rules. I have tested them recently (today and max 1 week old):

<details><summary>List with domains where you can find ads or left-overs</summary>

```
https://www.112achterhoek-nieuws.nl/
https://www.402online.com/

7sur7.be, ad.nl, bd.nl, bndestem.nl, destentor.nl, ed.nl, gelderlander.nl, hln.be, pzc.nl, tubantia.nl --> all on homepage

https://adultwebmaster.nl/forums/

https://www.afkortingen.nu/bios & other 2 on https://www.afkortingen.nu/

https://www.ajax1.nl/

https://www.alkmaarguardians.nl/

https://www.alle-tests.nl/ & https://www.bloovi.be/

https://www.alblasserdamsnieuws.nl/

https://allecijfers.nl/verkiezingsuitslagen/

1st: https://almere-nieuws.nl/p2000/
2nd: https://www.almere-nieuws.nl/nieuws/25595/mogelijk-verkeersconflict-voorafgaand-aan-schietpartij-in-tussen-de-vaarten/
3-5: https://www.almere-nieuws.nl/ (& [NSFW] https://www.sexjobs.nl/)

https://ans-online.nl/

https://www.apparata.nl/

https://autosport.nl/

1st: https://www.autowereld.nl/goedkope-occasions
2nd: https://www.autowereld.nl/

https://www.arenalokaal.nl/
https://www.dehavenloods.nl/
https://www.dekrantnieuws.nl/
https://www.destreekkrant.nu/
https://www.gooieneembode.nl/
https://www.hetkrantje-online.nl/
https://www.laardercourant.nl/
https://www.leiderdorpsweekblad.nl/
https://www.nnp.nl/
https://www.weespernieuws.nl/
https://www.zuidenvelder.info/

`.bannergroup`: all on homepage & https://www.dolcevia.com/nl/italie-magazine/lifestyle/4779-kerstavond-in-italie-het-feest-van-de-zeven-vissen

https://www.autowereld.nl/goedkope-occasions
https://www.autowereld.nl/
```
</details>

If I missed an URL then the element is on the homepage.

2. Screenshots

*To many to add*

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by JohnyP36** [comment]
I know I didn't follow the guidelines on the point that it's easier to address every issue separate, but this were to many.

File: DutchFilter/sections/specific.txt [hunk 1]
```diff
@@ -4,6 +4,39 @@
...
+adultwebmaster.nl##.main-promo-bn-responsive
+afkortingen.nu###banner_rectangle
+afkortingen.nu###banner_right
+afkortingen.nu###banner_top
+agconnect.nl##[class$="_ad-slot"]
+alblasserdamsnieuws.nl##.albla-adlabel
+alblasserdamsnieuws.nl##.albla-widget
+alblasserdamsnieuws.nl#?#.adv-link:not([href*="alblasserdamsnieuws.nl"]):upward(div[id^="albl"])
+alblasserdamsnieuws.nl#?#.sidebarbox > .widgettitle:has-text(Banner)
+alkmaarguardians.nl###footer
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:21]: Not even clickable. Maybe better to Annoyances-Self Promo.
```suggestion
```

File: DutchFilter/sections/specific.txt [hunk 2]
```diff
@@ -4,6 +4,39 @@
...
+alle-tests.nl,bloovi.be###advertising
+almere-nieuws.nl###containerx
+almere-nieuws.nl#?#.advertisement_block:upward(1)
+almere-nieuws.nl#?#.darksection:has-text(/Door onze partners/i):has(.postbox.owl-carousel)
+almere-nieuws.nl#?#.promocentered:upward(section)
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:26]: Too few visitors, leftovers are small
```suggestion
```

File: DutchFilter/sections/specific.txt [hunk 3]
```diff
@@ -4,6 +4,39 @@
...
+almere-nieuws.nl,sexjobs.nl##div[id^="banner-"]
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:27]: 0 px height
```suggestion
```

File: DutchFilter/sections/specific.txt [hunk 4]
```diff
@@ -4,6 +4,39 @@
...
+androidworld.nl,cinenews.be,handbal.nl,openrotterdam.nl,webmastersunited.com,wegdamnieuws.nl##.ads
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:28]: made another rule for androidworld.nl
```suggestion
androidworld.nl##.r89-desktop-billboard-atf
cinenews.be,handbal.nl,webmastersunited.com,wegdamnieuws.nl##.ads
```

File: DutchFilter/sections/specific.txt [hunk 5]
```diff
@@ -4,6 +4,39 @@
...
+ans-online.nl#?#.theiaStickySidebar > .widget:has-text(/Advertentie\b/)
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:29]: Blocks too much
```suggestion
ans-online.nl#?#.theiaStickySidebar > aside.penci_latest_news_widget > h4.widget-title:contains(Advertentie)
```

File: DutchFilter/sections/specific.txt [hunk 6]
```diff
@@ -4,6 +4,39 @@
...
+antilliaansdagblad.com,dolcevia.com,h2owaternetwerk.nl,lokaleomroepzeewolde.nl,radiolelystad.nl##.bannergroup
+apparata.nl#?#.widget:has(h2:has-text(/Partners/i))
+arenalokaal.nl,dehavenloods.nl,dekrantnieuws.nl,destreekkrant.nu,gooieneembode.nl,hetkrantje-online.nl,laardercourant.nl,leiderdorpsweekblad.nl,nnp.nl,weespernieuws.nl,zuidenvelder.info#?#.component__pubble-banner:has([advobject])
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:32]: It seems ok without `:has`
```suggestion
arenalokaal.nl,dehavenloods.nl,dekrantnieuws.nl,destreekkrant.nu,gooieneembode.nl,hetkrantje-online.nl,laardercourant.nl,leiderdorpsweekblad.nl,nnp.nl,weespernieuws.nl,zuidenvelder.info##.component__pubble-banner
```

File: DutchFilter/sections/specific.txt [hunk 7]
```diff
@@ -4,6 +4,39 @@
...
+arenalokaal.nl#?#.content-start > * > div[style]:has-text(/Externe links|Externe websites/i)
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:33]: AdGuard syntax
```suggestion
arenalokaal.nl#?#.content-start > * > div[style]:contains(/Externe links|Externe websites/i)
```

File: DutchFilter/sections/specific.txt [hunk 8]
```diff
@@ -4,6 +4,39 @@
...
+arenalokaal.nl,dehavenloods.nl,hetkrantje-online.nl,laardercourant.nl,leiderdorpsweekblad.nl#?#.component__pubble-banner:has([advobject]):upward(.bg-gray-100):has-text(/Uit de krant/i)
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:34]: Looks redundant
```suggestion
```

File: DutchFilter/sections/specific.txt [hunk 9]
```diff
@@ -4,6 +4,39 @@
...
+autosport.nl#?#.active.first:style(margin-top: 0px !important;)
```
Comment (Alex-302) [inline DutchFilter/sections/specific.txt:35]: It is their design of tabs
```suggestion
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by Alex-302** [comment]
Commited some changes directly to your PR with comments.

**Comment by JohnyP36** [comment]
> Commited some changes directly to your PR with comments.

Thanks for the feedback, that keeps me improving how I make rules with Adguard syntax.

**Review by JohnyP36 — COMMENTED** [review]
Thanks for correcting me.

**Pull request merged**